rt/features/logging/src/main/java/org/apache/cxf/ext/logging/osgi/Activator.java

```diff
@@ -91,6 +99,14 @@ public void updated(Dictionary config) throws ConfigurationException {
             if (logBinary != null) {
                 logging.setLogBinary(logBinary);
             }
+            
+            if (!sensitiveElementNames.isEmpty()) {
```

Comment:
@reta : wrong behaviour, condition should be removed to reflect if we comment out the property key